### ISSUE TITLE
WSL not working after clean install

### ISSUE THREAD

**Issue by rokatainment** [felixse/FluentTerminal#913]
I just did a clean install with wsl2 already installed.
Installed fluent terminal 0.7.5 via MS Store, then Ubuntu 20.04 LTS.
Ubuntu works fine in MS Terminal, but doesn't start in fluent terminal, no matter what i try. CMD and PS work fine.
I encountered that issue now on 2 of 2 machines. 
On the first machine for some reason it started working, but I don't know why exactly. I guess it had something to do with the installation of VS Code and/or some of the plugins for that.
All I get is this...
![fluent-terminal-wsl-problem](https://user-images.githubusercontent.com/4084466/115553527-cb5f6280-a2ad-11eb-987f-3cce5298ac49.png)

It says: "Session disconnected" Restart? Close?

**Comment by hanskokx** [comment]
What does the configuration for your WSL profile look like?

—
Hans Kokx
On 4/21/2021 8:29:27 AM, rokatainment ***@***.***> wrote:
I just did a clean install with wsl2 already installed.
Installed fluent terminal 0.7.5 via MS Store, then Ubuntu 20.04 LTS.
Ubuntu works fine in MS Terminal, but doesn't start in fluent terminal, no matter what i try. CMD and PS work fine.
I encountered that issue now on 2 of 2 machines.
On the first machine for some reason it started working, but I don't know why exactly. I guess it had something to do with the installation of VS Code and/or some of the plugins for that.
All I get is this...
[fluent-terminal-wsl-problem] [https://user-images.githubusercontent.com/4084466/115553527-cb5f6280-a2ad-11eb-987f-3cce5298ac49.png]
It says: "Session disconnected" Restart? Close?

**Comment by rokatainment** [comment]
The whole configuration is pretty much default. I haven't changed anything, yet.

![grafik](https://user-images.githubusercontent.com/4084466/115676188-a0c5e600-a34f-11eb-82ea-4e4d22fa2e5d.png)

However I tried it with ConPty enabled and disabled: same same.

One interesting thing is, that it works with Windows Terminal out of the box. So i don't think it's an WSL issue.

**Comment by rokatainment** [comment]
Okay, I found the solution. As I integrated WSL in IntelliJ it didn't start either. It kept working fine for VS Code and Windows Terminal. That seemed suspicious to me. So I rechecked my wsl configs and it turned out, that the default wsl distribution was changed back to docker-data. Though I set it to Ubuntu-20.04. It could have been a docker update or some changes I made there that made this switch happen under the hood.

However, setting it back to Ubuntu was the solution. Now I've got my nice lil fluent terminal back. :-)

![grafik](https://user-images.githubusercontent.com/4084466/115694915-9d3b5a80-a361-11eb-9a0e-1b1ec24ca470.png)
